Bottom: 7b5a5da32ab6fe32f2fa6ec08692b7d853fefef9
Top:    d02ca54f77213fdac666f0bcfb15f3a0d6fb0ee9
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-13 17:31:17 +0000

Refresh of make-poly_copy1-a-static

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 663dd11..c1c15dd 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -2979,7 +2979,7 @@ poly_Copy1 (POLYAREA * dst, const POLYAREA * src)
   for (cur = src->contours; cur != NULL; cur = cur->next)
     {
       if (!poly_CopyContour (last, cur))
-	return FALSE;
+        return FALSE;
       r_insert_entry (dst->contour_tree, (BoxType *) * last, 0);
       last = &(*last)->next;
     }
